app/src/main/java/org/stepik/android/view/step_quiz_fullscreen_code/ui/delegate/CodeQuizFormFullScreenDelegate.kt

```diff
@@ -0,0 +1,170 @@
+package org.stepik.android.view.step_quiz_fullscreen_code.ui.delegate
+
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import android.view.ViewGroup
+import android.widget.RelativeLayout
+import kotlinx.android.synthetic.main.fragment_step_quiz_code_fullscreen_playground.view.*
+import kotlinx.android.synthetic.main.layout_step_quiz_code.view.codeStepLayout
+import org.stepic.droid.R
+import org.stepic.droid.code.util.CodeToolbarUtil
+import org.stepic.droid.model.code.extensionForLanguage
+import org.stepic.droid.persistence.model.StepPersistentWrapper
+import org.stepic.droid.ui.adapters.CodeToolbarAdapter
+import org.stepic.droid.ui.util.inflate
+import org.stepic.droid.ui.util.setOnKeyboardOpenListener
+import org.stepic.droid.util.DpPixelsHelper
+import org.stepik.android.model.Reply
+import org.stepik.android.presentation.step_quiz.StepQuizView
+import org.stepik.android.presentation.step_quiz.model.ReplyResult
+import org.stepik.android.view.base.ui.interfaces.KeyboardExtensionContainer
+import org.stepik.android.view.step_quiz.resolver.StepQuizFormResolver
+import org.stepik.android.view.step_quiz.ui.delegate.StepQuizFormDelegate
+import org.stepik.android.view.step_quiz_code.mapper.CodeStepQuizDetailsMapper
+import org.stepik.android.view.step_quiz_code.mapper.CodeStepQuizFormStateMapper
+import org.stepik.android.view.step_quiz_code.model.CodeDetail
+import org.stepik.android.view.step_quiz_code.model.CodeStepQuizFormState
+import org.stepik.android.view.ui.delegate.ViewStateDelegate
+import ru.nobird.android.ui.adapterssupport.DefaultDelegateAdapter
+
+class CodeQuizFormFullScreenDelegate(
```

Comment:
As I remember we want to remove code quiz actions on small code editor and open fullscreen by clicking on small editor. It will be better to reuse the same FormDelegate from small one.